about.html

```diff
@@ -20,23 +20,40 @@
                     <a href="index.html"><img src="images/logo 1.svg" class="logo-img" alt="logo"></a>
                 </div>
                 <div class="navigation-header">
-                    <a class="navigation-link-text" href="index.html">Home</a>
-                    <a class="navigation-link-text" href="about.html">About</a>
-                    <a class="navigation-link-text" href="contact.html">Contact</a>
+                    <div class="icons-navigation-text">
+                        <a class="navigation-link-text" href="index.html">Home</a>
+                        <a class="navigation-link-text" href="about.html">About</a>
+                        <a class="navigation-link-text" href="contact.html">Contact</a>
+                    </div>
                     <a class="navigation-link" href="https://github.com/mikaelelias"><img src="images/Vector.svg"
                             alt="logo-gthub"></a>
-                    <a class="navigation-link" href=""><img src="images/ant-design_twitter-circle-filled.svg"
+                    <a class="navigation-link" href="https://twitter.com/MikaelElias9"><img src="images/ant-design_twitter-circle-filled.svg"
                             alt="logo-twitter"></a>
                     <a class="navigation-link" href="https://www.linkedin.com/in/mikael-elias-64973b218/"><img
                             src="images/entypo-social_linkedin-with-circle.svg" alt="logo-linkedin"></a>
                     <a class="navigation-link-bi-list">
                         <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor"
-                            class="bi bi-list" viewBox="0 0 16 16" alt="bi-list">
+                        class="bi-bi-list open-icon" viewBox="0 0 16 16" alt="bi-list">
                             <path fill-rule="evenodd"
                                 d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
                         </svg></a>
+
+                        <div class="mobile-icon-menu">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                                class="bi-bi-x-lg close-icon" viewBox="0 0 16 16">
+                                <path
+                                    d="M2.146 2.854a.5.5 0 1 1 .708-.708L8 7.293l5.146-5.147a.5.5 0 0 1 .708.708L8.707 8l5.147 5.146a.5.5 0 0 1-.708.708L8 8.707l-5.146 5.147a.5.5 0 0 1-.708-.708L7.293 8 2.146 2.854Z" />
+                            </svg>
```

Comment:
Eu aconselho a gente copiar esse trecho de código e colocas num arquivo `.svg` separado e usar uma tag `img` importando a imagem como `SVG`, ex:
```suggestion
                            <img src="images/novo_icone.svg"></img>
```

Reply:
Não sabia que dava pra fazer isso, bem mais eficiente. 
Tá feito.